marklogic-data-hub-central/ui/src/components/entities/merging/merging-step-detail/merging-step-detail.tsx

```diff
@@ -57,11 +59,26 @@ const MergingStepDetail: React.FC = () => {
         if (Object.keys(curationOptions.activeStep.stepArtifact).length !== 0) {
             const mergingStepArtifact: MergingStep = curationOptions.activeStep.stepArtifact;
             setMergingStep(mergingStepArtifact);
+            retrieveCalculatedMergingInfo(mergingStepArtifact);
         } else {
             history.push('/tiles/curate');
         }
     }, [JSON.stringify(curationOptions.activeStep.stepArtifact)]);
 
+    const retrieveCalculatedMergingInfo = async (mergingStepArtifact: MergingStep) => {
```

Comment:
Since "Activity" is used elsewhere, I think this should be "..MergingActivity" instead of "..MergingInfo".

Reply:
Any reason to keep this "Info" instead of "Activity"?